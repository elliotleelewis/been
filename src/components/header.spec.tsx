import { describe } from 'vitest';

describe('Header', () => {
	// TODO
});
