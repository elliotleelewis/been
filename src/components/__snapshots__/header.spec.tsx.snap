// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > should render 1`] = `
<DocumentFragment>
  <div
    class="items-center justify-center bg-primary text-white dark:bg-zinc-950 dark:text-primary flex sm:hidden"
  >
    <h1
      class="p-3 text-xl font-bold tracking-wide"
    >
      been
    </h1>
  </div>
</DocumentFragment>
`;
